userguide/sharing.rst

```diff
@@ -1792,8 +1792,8 @@ for iSNS requests is *5* seconds.
    |                                 |                              |                                                                                           |
    |                                 |                              |                                                                                           |
    +=================================+==============================+===========================================================================================+
-   | Base Name                       | string                       | See the "Constructing iSCSI names using the iqn. format" section of :rfc:`3721`           |
-   |                                 |                              | if unfamiliar with this format.                                                           |
+   | Base Name                       | string                       | Lowercase alphanumeric characters plus dot (.), dash (-), and colon (:) are allowed.      |
```

Comment:
There's a number issue with the sentence "Lowercase alphanumeric characters plus dot (.), dash (-), and colon (:) are allowed." -> Lowercase alphanumeric characters with dots (.), dashes (-), and colons (:) are allowed.

Reply:
I think this is clear, and simpler with the singular versions.